src/database/migrate.ts

```diff
@@ -36,11 +36,15 @@ export default async function migrate() {
 
   try {
     application.logger.log('Starting synchronize + migrations.');
+    application.logger.debug('Synchronize...');
     await application.connection.synchronize();
+    application.logger.debug('Fake migrations...');
     await application.connection.runMigrations({ transaction: 'all', fake: true });
+    application.logger.debug('Revert last migration...');
     await application.connection.undoLastMigration({ transaction: 'all' });
+    application.logger.debug('Run last migration...');
     await application.connection.runMigrations({ transaction: 'all' });
-    await application.connection.close();
+    await application.connection.destroy();
```

Comment:
Is dit waarom hij bleef hangen????

Reply:
![image](https://github.com/user-attachments/assets/06ca3ccd-8c4d-494f-80e7-fd2b23a245a9)
het lijkt iets te doen